backend/router.js

```diff
@@ -6,9 +6,12 @@ const Controller = require("./Controller.js");
 router.get("/token/:token/user", Controller.getUser)
 
 //define post request for making a new group
-router.post("/group/create", Controller.createGroup);
+router.post("/groups/create", Controller.createGroup);
```

Comment:
personally prefer the group non-plural route name as it is more indicative (single group being created not multiple). but this is a matter of opinion